oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -643,6 +643,7 @@ impl Runtime {
         &self,
         node_id: NodeId,
         label: &Label,
+        runtime_weak: Weak<Runtime>,
```

Comment:
I wonder if it might be neater to move this to have signature `fn channel_create(self: &Arc<Self>, node_id: NodeId, label: &Label) -> ...` and then use `Arc::downgrade(self)` below &ndash; WDYT?

Reply:
Why not, that works for me. :)